doc/classes/AudioStreamPlayer.xml

```diff
@@ -56,6 +56,9 @@
 		<member name="bus" type="StringName" setter="set_bus" getter="get_bus" default="&amp;&quot;Master&quot;">
 			Bus on which this audio is playing.
 		</member>
+		<member name="max_polyphony" type="int" setter="set_max_polyphony" getter="get_max_polyphony" default="1">
+			The maximum number of sounds this node can play at the same time.
```

Comment:
The behavior that happens when the polyphony limit is exceeded should be documented. Is it a FIFO/LIFO mechanism or something less deterministic? In other words, do the oldest sounds always get cut off first?

This comment also applies to AudioStreamPlayer2D and AudioStreamPlayer3D.

Reply:
I'll make sure to document that before taking this out of draft state. Right now it's FIFO, but would it make sense to switch to LIFO? Or make it configurable?

Reply:
FIFO is likely the behavior that makes the most sense. The oldest sounds are generally the ones that have the lowest audible impact when they are suddenly cut off.

Reply:
Fixed! Thanks.

Reply:
It is very important for me to retain the ability to have new sounds cause the old sound to stop (be cut off) so the new sound can start immediately like was the behavior in Godot 3. Can I still do that with this?

Reply:
Yes, the default Polyphony value is set to 1 and acts like in 3.x.

Reply:
Yup, just like Calinou said, you only get this new behavior when you configure an audio stream player to work this way by changing `Max Polyphony` to a value other than 1. It's mean to just make some use-cases easier to deal with, not to fundamentally change the way audio stream players work.